### PR TITLE
Implemented create button for lobbies

### LINKED ISSUE #256: Implement a create view for lobbies
**Goal:** We need users to be able to create lobbies. This will be the Create page of the CRUD pages for lobbies.

### PR DESCRIPTION
**Pull request by jiyeongchloe**
We needed users to be able to create lobbies, so we added a "create view" that allows users to submit a form to create lobbies (but they do not have access to `match_status`, `created_by`, and `lobby_created` attributes---these are filled in automatically)

### CODE REVIEW

**Comment by nugerhard** [comment]
Doesn't seem like merging with dev fixed the run-test failure. Looks like a migrations issue.

**Comment by nugerhard** [comment]
Great. For reference: src/chigame/games/migrations/0014_alter_game_year_published.py deleted, as it was conflicting with migration in dev.

**Review by nugerhard — APPROVED** [review]
Works for me, and I agree with the set of fields we are displaying to users. Eventually, game and game mod status may do something more interesting to the lobby creation page, but that's a task for a future matches team. Nice job!

File: src/templates/games/lobby_form.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h2>Create a new Lobby</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Save</button>
+  </form>
+{% endblock body %}
```
Comment (elizabethli31) [inline src/templates/games/lobby_form.html]: I'm getting a syntax error, but it's fixed by changing this line to `endblock content` or simply `endblock`

**Review by elizabethli31 — CHANGES_REQUESTED** [review]
Good functionality and good work! Minor syntax error (see comment).

File: src/templates/games/lobby_form.html (hunk 1 above)
Comment (xylucien) [reply src/templates/games/lobby_form.html]: thank you! fixed

**Review by elizabethli31 — APPROVED** [review]
LGTM! Create form is nice and clean, and everything is working properly now.

**Comment by jiyeongchloe** [comment]
**Closing Comment:**
We instantiated a Create button/link that allows users to create lobbies. There were no restrictions placed on who could create lobbies because we want all users to be able to create them. Users' access to certain attributes were restricted, however, as we don't want users to be able to manually change settings like the host of the lobby and match status.

**Comment by elizabethli31** [comment]
Issue Score: Excellent

Comments:
Much needed functionality - nice work on this PR

**Pull request merged**